pagopa/io-app
# [IOAPPX-404] Remove legacy components related to the previous UI

## Short description
This PR removes legacy components related to the previous UI, including local typographic styles. This move is necessary because we need to get rid of the legacy typography to avoid typographic inconsistencies [once the `IOText` component is introduced](https://github.com/pagopa/io-app/pull/6076).

## List of changes proposed in this pull request
- Remove the following typographic styles:
  - `Body`
  - `H1`
  - To be continued…
- Remove the following legacy components:
  - `AdviceComponent`
  - `RawAccordion`
  - `DetailedListItemComponent`
- Remove the following pages:
  - `DSLegacyAccordion`

## How to test
Navigate through the app and check that everything is OK. Keep in mind that some slight changes are normal because there isn't a perfect match between legacy and new typographic styles.
